netctl/netctl.go

```diff
@@ -237,23 +235,18 @@ func listNetworks(ctx *cli.Context) {
 	} else {
 		writer := tabwriter.NewWriter(os.Stdout, 0, 2, 2, ' ', 0)
 		defer writer.Flush()
-		writer.Write([]byte("Tenant\tNetwork\tPublic\tEncap\tSubnet\tGateway\n"))
-		writer.Write([]byte("------\t-------\t------\t-----\t------\t-------\n"))
+		writer.Write([]byte("Tenant\tNetwork\tEncap\tTag\tSubnet\tGateway\n"))
```

Comment:
+1.
I would like the headings to be clear even when table had no values. "Encap Type" and "Packet Tag" are better, as they are also more explicit about what to expect in those columns.

Just as side, do we need same for the corresponding cli flags as well? Right now they are `encap` and `pkt-tag` but may be they should also be `encap-type` and `pkt-tag`. But upto you.